docs/source/generate.rst

```diff
@@ -41,16 +44,21 @@ Node
         doc.render()
     }
     catch (error) {
-        // The error thrown here contains additional information when logged with JSON.stringify (it contains a properties object containing all suberrors).
-        function replaceErrors(key, value) {
-            if (value instanceof Error) {
-                return Object.getOwnPropertyNames(value).reduce(function(error, key) {
-                    error[key] = value[key];
-                    return error;
-                }, {});
-            }
-            return value;
+        errorHandler(error);
+    }
+
+    // The error thrown here contains additional information when logged with JSON.stringify (it contains a properties object containing all suberrors).
+    function replaceErrors(key, value) {
```

Comment:
Can you put `replaceErrors` and `errorHandler` before the line : 

```
//Load the docx file as a binary
    var content = fs
```

(The same for the other examples), in order to avoid having functions interlaced with the main code.

Reply:
I will refactor it.